packages/primitives/package.json

```diff
@@ -3,11 +3,18 @@
   "version": "9.2.9",
   "main": "dist/index.cjs.js",
   "module": "dist/index.esm.js",
+  "types": "types/index.d.ts",
   "files": [
     "src",
-    "dist"
+    "dist",
+    "types"
   ],
+  "scripts": {
+    "test:typescript": "dtslint types"
+  },
   "dependencies": {
+    "@types/react-native": "^0.56.19",
+    "@types/react-primitives": "^0.6.2",
```

Comment:
Likewise, deps on `emotion`.

Reply:
Type for this depends on `emotion`. If this package should not depends on it, then type also should.

Reply:
I'd prefer if this doesn't depend on emotion, create-emotion or create-emotion-styled for the types since it doesn't depend on any of them in the code.

Reply:
I'll decouple them fully and update the PR :)